### DISCUSSION TITLE
Add SURFconext as authentication option

### DISCUSSION THREAD

**Discussion by JeltevanBoheemen** [UUDigitalHumanitieslab/I-analyzer#1614] [Ideas]
See https://wiki.surfnet.nl/display/surfconextdev/Connect+in+5+Steps for implementation details.

Advantages:
- Researchers of other institutions do not need to register for I-Analyzer
- Allow corpus access based on being a member of institution
- We would be _conexted_ :sunglasses:

Disadvantages:
- The gain is minimal with current corpora. All corpora that require Solis login are limited to UU staff only, which would not improve when logging in through SURFconext

**Comment by lukavdplas** [comment]
I think this is a duplicate of #1581 ?

**Reply by JeltevanBoheemen** [reply]
Yep!
